app/views/layouts/_list_grid.html.haml

```diff
@@ -73,5 +73,8 @@
                                       :alt     => cell[:title]}
                 = cell[:text]
             - else
-              = image_tag(cell[:image], :alt => cell[:title], :title => cell[:title]) if cell[:image]
+              -# :icon supersedes :image
+              = image_tag(cell[:image], :alt => cell[:title], :title => cell[:title]) if cell[:image] && cell[:icon].nil?
+              - if cell[:icon].present?
```

Comment:
Yeah, it's a three-state.. we can have icon, image, both or neither .. and we want to prefer the icon if we have both. Aand you can't use image_tag for the icon.